centos_ci.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Output command before executing
+set -x
+
+# Exit on error
+set -e
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+if [ -e "jenkins-env" ]; then
+  cat jenkins-env \
+    | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+    | sed 's/^/export /g' \
+    > ~/.jenkins-env
+  source ~/.jenkins-env
+fi
+
+# We need to disable selinux for now, XXX
+/usr/sbin/setenforce 0
+
+# Get all the deps in
+yum -y install \
+  gcc \
+  make \
+  git \
+  curl
+
+# Install Go 1.7
+curl -LO https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz
+tar -xf go1.7.linux-amd64.tar.gz
+mv go /usr/local
+
+# Setup GOROOT
+export GOROOT=/usr/local/go
+# Setup GOPATH
+mkdir $HOME/gopath $HOME/gopath/src $HOME/gopath/bin $HOME/gopath/pkg
+export GOPATH=$HOME/gopath
+export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
+
+echo $GOPATH
+echo $GOROOT
+
+# Setup minishift repo
+mkdir -p $GOPATH/src/github.com/minishift/minishift
+cp -r `ls` $GOPATH/src/github.com/minishift/minishift/
+cd $GOPATH/src/github.com/minishift/minishift/
+
+# Glide setup
+GLIDE_OS_ARCH=`go env GOHOSTOS`-`go env GOHOSTARCH`
```

Comment:
Instead of all this stuff below have you tried: "curl https://glide.sh/get | sh"